devel/ci/Dockerfile-f28

```diff
@@ -43,6 +43,7 @@ RUN dnf install --disablerepo rawhide-modular -y \
     python3-munch \
     python3-openid \
     python3-pillow \
+    python3-psycopg2 \
```

Comment:
Was this needed to get the tests to pass?

If so, we should probably mock the setting in the unit tests so they they keep using sqlite, otherwise we won't be able to run the unit tests downstream very easily. Also, I expect that the unit tests are probably slower when run with PG.

What do you think?

Reply:
As far as I remember I added `python3-psycopg2` because of failing integration tests.
I thing that unit tests are still using sqllite:
https://github.com/fedora-infra/bodhi/blob/27ea7fb92c37beb45da66f5a7b0b50d5e8a0e75d/bodhi/tests/server/base.py#L38

Reply:
I realized that there was a patch that does this already on develop. I'd forgotten that the tests started failing on develop a while ago. My guess is that you based your patch on a commit that needed psycopg2, and therefore that it's actually not related to the change you are making, which is a coincidence.

After rebasing your patch, I see that these changes aren't even part of it anymore since they are already in develop.